app/page.tsx

```diff
@@ -35,11 +35,11 @@ export default function IndexPage() {
             <Link
               target="_blank"
               rel="noreferrer"
-              href={siteConfig.links.webxdao_gh}
+              href="https://guild.xyz/webx-dao"
```

Comment:
can you please move this hardcode url to the site config file?

Reply:
Ah... yeah, I missed this. I just :scream: with the :european_castle: